.github/workflows/create-scheduled-release.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: "🕸️ Get changed metadata matrix"
         id: set-matrix
         run: |
-          LATEST_TAG=$(git tag --list | sort -V | tail -1)
+          LATEST_TAG=$(git tag --list | grep -v latest | sort -V | tail -1)
```

Comment:
This additional step skips `latest` tag as potential base for next standard release tag name (as we always bump patch version in the version string, we want to do so from the latest **standard** release).

Reply:
Maybe write that comment in the file?